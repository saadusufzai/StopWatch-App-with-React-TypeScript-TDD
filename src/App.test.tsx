import React from 'react';
import { shallow } from 'enzyme';
import App from './components/AppComponent/App';
test('renders the component', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});