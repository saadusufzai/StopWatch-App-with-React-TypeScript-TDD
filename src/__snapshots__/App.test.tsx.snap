// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
 <div className='app-container'>
     <Timer/>
   </div>

`;
